src/reflow/mod.rs

```diff
@@ -64,26 +64,40 @@ fn reflow_inner<'s>(
     let mut gluon = Gluon::new(s_absolute, max_line_width, indentations);
     gluon.add_unbreakables(unbreakables);
 
-    let prefix = match variant {
-        CommentVariant::Commonmark | CommentVariant::MacroDocEq => "",
-        CommentVariant::TripleSlash => " ",
+    let (prefix, comment) = match variant {
+        CommentVariant::Commonmark | CommentVariant::MacroDocEq => ("", ""),
+        CommentVariant::TripleSlash => (" ", "///"),
+        CommentVariant::DoubleSlashEM => (" ", "//!"),
         CommentVariant::Unknown => return None,
     };
+    let mut reflow_applied = false;
+    let mut lines = s.lines();
+    let mut acc: String = if let Some((_, content, _)) = gluon.next() {
```

Comment:
Could you elaborate on this?

Reply:
I agree, the check if a reflow happend is not really intuitive. It compares each line in the original string `s` with the result of the reflow, and right while I'm writing this comment, I see a bug. The comparison needs to happen on the absolute string since only that is fed into the `gluon`. I'll test and fix this and also add some comments :)

Reply:
Some more context: now the lines in the input to the `gluon` is compared to its output. If nothing changed, there was no reflow. Does that make more sense?

On the doc comment above: right now, the chunk shown by you would be reflown to `#[ doc = r#"a b c"]` if the the `max_line_width > 19 + indentation`. Is this not desired? I added a test for this exact case.

Reply:
another point about doc comments: we should keep the indentation of the second and following lines:

```
#[ doc = "First line
               Second line blabla"]
// should stay like this:
#[ doc = "First line Second
               line blabla"]
```
right?